pkg/local_object_storage/writecache/init.go

```diff
@@ -12,11 +13,55 @@ import (
 	"go.uber.org/zap"
 )
 
-func (c *cache) initFlushMarks() {
+func (c *cache) initFlushMarks(async bool) {
+	if !async {
+		c.fsTreeFlushMarkUpdate()
+		c.dbFlushMarkUpdate()
+		c.inited = true
```

Comment:
Can you add a comment about why we don't need a mutex here?

Reply:
added but depends on async flag discussion (seems like it could be even removed)